### ISSUE TITLE
How to create proper desktop launchers for remoteapps?

### ISSUE THREAD

**Issue by relink2013** [FreeRDP/FreeRDP#4582]
How can we create proper functioning desktop launchers for our remote apps? We used to have "Winconn" but it has been abandoned and doesn't work well on newer versions of linux. 

I currently have some launchers that I made that just point to a .sh file and that works ok, except the programs spawn seperate "X11" icons on my launcher, needless to say it makes it difficult to distinguish between multiple remoteapps when they all have the same X11 icon. 

I know there has to be a way to do this, any help would be appreciated, and if possible it could be added to the readme. Im sure this is a pretty common use case.

**Comment by relink2013** [comment]
I think what I really need to beable to do is to set the "WM_CLASS" of my remote app, but how do I do that? I see that it was possible with the "old" syntax, but I dont see how to do it with the new.

**Comment by akallabeth** [comment]
https://github.com/FreeRDP/FreeRDP/wiki/BugReporting
btw: `.desktop` files should suffice

**Comment by relink2013** [comment]
Oh it does, except that when launching the `.desktop` file I do not get the apps icon on my launcher, I get a generic "X" icon for all remote apps. Once I have about 3 or 4 remote apps going at the same time its gets very confusing which one is which. 

However I did find the solution. in the freeRDP command I can add `/wm-class <class name>`
and then in the `.desktop` file I add the line `StartupWMClass=<class name>` and that solves the problem. 

That `/wm-class <class name>` option really needs to be added to the Command line documentation. I couldnt find that option anywhere on this website, I actually found it on the debian.org page located [here](https://manpages.debian.org/testing/freerdp-x11/xfreerdp.1.en.html)
